packages/date-picker/src/FilterDateRangePicker/components/DateRangeValidationMessage/DateRangeValidationMessage.tsx

```diff
@@ -36,11 +38,10 @@ export const DateRangeValidationMessage = ({
     if (status.dateEnd === status.dateStart) {
       return (
         <FieldMessage
-          id={id}
           message={
             <ul className={styles.fieldMessageList}>
-              <li>{validationMessage.dateEnd}</li>
-              <li>{validationMessage.dateStart}</li>
+              <li id={dateStartId}>{validationMessage.dateEnd}</li>
+              <li id={dateEndId}>{validationMessage.dateStart}</li>
```

Comment:
Could you switch these around please? My brain expects the Start one to come first 😂 

<img width="672" alt="image" src="https://user-images.githubusercontent.com/25891850/222293421-df22e6ed-84c0-4c89-97d2-2172f6499f33.png">